.github/workflows/chronograf-cypress-tests.yaml

```diff
@@ -79,6 +79,7 @@ jobs:
             apt update
             apt install -y build-essential
             yarn install --frozen-lockfile
+            yarn run cypress install
```

Comment:
I added this line, because the tests failed on
```
/chronograf/node_modules/.bin/cypress run --browser chrome --config baseUrl=http://host.docker.internal:8888/ --reporter junit --reporter-options mochaFile=cypress/results/results-[hash].xml
No version of Cypress is installed in: /root/.cache/Cypress/8.4.1/Cypress

Please reinstall Cypress by running: cypress install
```